### DISCUSSION TITLE
Running motion correction from morphology channel and applying shifts to functional channel

### DISCUSSION THREAD

**Discussion by pbl007** [nel-lab/mesmerize-core#286] [Q&A] [answered]
Hi
I am wondering if it is possible to apply motion correction shifts computed from a morphological channel to a functional channel. For example, running motion correction on a channel containing  tdTomato expression in cell soma and applying the shifts to the calcium imaging channel.
Thanks in advance,
Pablo

**Comment by kushalkolar** [comment]
This would be a caiman question and I think it's possible but not trivial, you can ask at the caiman repo.

**Comment by kushalkolar** [comment, marked as answer]
@pbl007 I think that this is easier with jnormcorre and from your other issue you seem to have a decent GPU so it could be a good alternative to caiman, it's under active development so you're likely to get assistance on how to do this: https://github.com/apasarkar/jnormcorre 

The developer (Amol) is also working to create fastplotlib-based visualizations like we've done with mesmerize so you'll get the visualization advantages as well.
